### PR TITLE
Upgrade version to 2.3.0

### PR DESCRIPTION
**Pull request by rainyfly**
1.  补充Readme说明文档
2. 修复后端支持--model的bug
3. 升级版本到2.3.0

### CODE REVIEW

File: README_CN.md [hunk 1]
```diff
@@ -337,12 +337,18 @@ value: 3.1297709941864014
 
 ### Graph
 
-一键可视化模型的网络结构。可查看模型属性、节点信息、节点输入输出等，并支持节点搜索，辅助用户快速分析模型结构与了解数据流向。
+一键可视化模型的网络结构。可查看模型属性、节点信息、节点输入输出等，并支持节点搜索，辅助用户快速分析模型结构与了解数据流向。当前分为动态图和静态图两个版本。
```
Comment (YixinKristy) [inline README_CN.md]: 一键可视化模型的网络结构。可查看模型属性、节点信息、节点输入输出等，并支持节点搜索，辅助用户快速分析模型结构与了解数据流向，覆盖动态图与静态图两种格式。

File: README.md [hunk 2]
```diff
@@ -315,10 +315,18 @@ Developers can compare multiple experiments by specifying and uploading the path
 
 ### Graph
 
-**Graph** enables developers to visualize model structures by only one click. Moreover, **Graph** allows developers to explore model attributes, node information, node input and output. aiding them analyze model structures quickly and understand the direction of data flow easily.
+**Graph** enables developers to visualize model structures by only one click. Moreover, **Graph** allows developers to explore model attributes, node information, node input and output. aiding them analyze model structures quickly and understand the direction of data flow easily. Currently we provide two versions to visualize dynamic and static model graph individually.
```
Comment (YixinKristy) [inline README.md]: **Graph** enables developers to visualize model structures by only one click. Moreover, **Graph** allows developers to explore model attributes, node information, node input and output, aiding them analyze model structures quickly and understand the direction of data flow easily. Additionally, **Graph** supports the visualization of dynamic and static model graph respectively.

File: README_CN.md (hunk 1 above)
Comment (rainyfly) [reply README_CN.md]: Done

**Review by nepeplwu — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (rainyfly) [reply README.md]: Done

**Pull request merged**